test/protocol/SellerHandlerTest.js

```diff
@@ -2181,6 +2208,33 @@ describe("SellerHandler", function () {
           );
```

Comment:
Just above this on line 2200, there's a test case "addresses are the zero address". There's something not quite right with this test case. It looks like it's meant to check if the INVALID_ADDRESS revert gets triggered when all the addresses are zero. However, it doesn't set the treasury address to zero. It DOES set the admin address to zero, but it also passes in a valid authToken (not emptyAuthToken). So a zero admin address would be valid.  A different revert happens if the admin address/auth token combo is not correct (ADMIN_OR_AUTH_TOKEN). I think to make this a valid use case, the line 

`seller.admin = ethers.constants.AddressZero;` on line 2202 should be changed to 
`seller.treasury = ethers.constants.AddressZero;`

There doesn't seem to be a corresponding test case in the createSeller Revert Reasons context

Reply:
The problem in the test case starting on line 2200 has been fixed. There is still no corresponding "addresses are the zero address" test case for createSeller. To be consistent, it should go just above  the "Operator address is zero address" test case on line 909.

Reply:
sorry I overlooked your comments